Modules/ManualAssessment/classes/class.ilManualAssessmentMemberGUI.php

```diff
@@ -16,171 +16,211 @@
  *
  * @author Denis Klöpfer <denis.kloepfer@concepts-and-training.de>
  */
-class ilManualAssessmentMemberGUI {
-		protected $notificator;
-
-		public function __construct($members_gui ,$a_parent_gui, $a_ref_id) {
-			$this->notificator = new ilManualAssessmentPrimitiveInternalNotificator();
-			global $ilCtrl, $tpl, $lng, $ilUser, $ilTabs;
-			$this->ctrl = $ilCtrl;
-			$this->members_gui = $members_gui;
-			$this->parent_gui = $a_parent_gui;
-			$this->object = $a_parent_gui->object;
-			$this->ref_id = $a_ref_id;
-			$this->tpl =  $tpl;
-			$this->lng = $lng;
-			$this->ctrl->saveParameter($this,'usr_id');
-			$this->examinee = new ilObjUser($_GET['usr_id']);
-			$this->examiner = $ilUser;
-			$this->setTabs($ilTabs);
-			$this->member = $this->object->membersStorage()
-								->loadMember($this->object, $this->examinee);
-	}
-
-	public function executeCommand() {
-		$edited_by_other = $this->targetWasEditedByOtherUser($this->member);
-		$read_permission = $this->object->accessHandler()->checkAccessToObj($this->object,'read_learning_progress');
-		$edit_permission = $this->object->accessHandler()->checkAccessToObj($this->object,'edit_learning_progress');
-		if(!$read_permission && !$edit_permission) {
-			$a_parent_gui->handleAccessViolation();
-		}
+class ilManualAssessmentMemberGUI
+{
+	protected $notificator;
+
+	public function __construct($members_gui, $a_parent_gui, $a_ref_id)
+	{
+		$this->notificator = new ilManualAssessmentPrimitiveInternalNotificator();
+		global $ilCtrl, $tpl, $lng, $ilUser, $ilTabs;
+		$this->ctrl = $ilCtrl;
+		$this->members_gui = $members_gui;
+		$this->parent_gui = $a_parent_gui;
+		$this->object = $a_parent_gui->object;
+		$this->ref_id = $a_ref_id;
+		$this->tpl =  $tpl;
+		$this->lng = $lng;
+		$this->ctrl->saveParameter($this, 'usr_id');
+		$this->examinee = new ilObjUser($_GET['usr_id']);
```

Comment:
Please leave according tickets in Mantis. I currently don't see directly bad implications, but the style certainly is questionable.